webdriver-spec.html

```diff
@@ -3220,35 +3222,26 @@ <h2 id="element-location-strategies">Element Location Strategies</h2>
 
   <p>Otherwise, return <var>element result</var>.
 
- <li><p>If <a>element</a> <a>is stale</a>,
+ <li><p>If <var>element</var> <a>is stale</a>,
   return <a>error</a> with <a>error code</a> <a>stale element reference</a>.
 
- <li><p>Let <var>name</var> be equal to "name" from <a>match a request</a>.
-   if <var>name</var> is undefined return <a>error</a> with <a>error code</a>
-   <a>invalid argument</a>.
-  <li>Let <var>result</var> be the result of the first appropriate step below:
-
-   <dl class=switch>
-     <dt><p>If <var>name</var> is a
-       <a href='https://html.spec.whatwg.org/#boolean-attribute'>boolean attribute</a>:
-     <dd><p>The first matching statement if the element
-       <a href=https://dom.spec.whatwg.org/#dom-element-hasattribute>has the attribute</a>:
-       <dl class=switch>
-         <dt><code>true</code>
-         <dd><p>'true'.
-         <dt>Otherwise
-         <dd><p><code>Null</code>.
-      </dl>
-      <dt><p>Otherwise:
-      <dd><p>The result of <a href=https://dom.spec.whatwg.org/#concept-element-attributes-get-by-name>
-          getting the attribute by name</a>.
-   </dl>
-   <li><p>Let <var>body</var> be a JSON Object
-    with the "<code>value</code>" member set to <var>result</var>.
-
-   <li><p>Return <a>success</a> with data <var>body</var>.
+ <li><p>If <var>name</var> is undefined,
+  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>attribute</var> be the result of
+  <a href=https://dom.spec.whatwg.org/#concept-element-attributes-get-by-name>getting an attribute</a>
+  given <var>name</var> and <var>element</var>.
+
+ <li><p>Let <var>result</var> be the value of <var>attribute</var>
```

Comment:
Per my comment on the Wires bug, I don't actually see how to implement the currently specified behaviour without a list of boolean attributes in the implementation. That's because it isn't possible to tell whether a specific content attribute is actually a boolean attribute through DOM APIs.